### ISSUE TITLE
Adding a CMD param in the middle of arguments array breaks downstream

### ISSUE THREAD

**Issue by joostschriek** [terraform-google-modules/terraform-google-gcloud#163]
### TL;DR

Adding a parameter to the kubectl_wrapper in the _middle_ (or, _position 6 of >6_) of the CMD execution and then releasing this as a quick _minor_ update might break things downstream where a lot of implementations of [this module have the version contraint `~>3.1`](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/3f27bc20bf5ee81ccda862307fa8c9624357ec93/modules/asm/main.tf#L60) like googles own k8s modules. 

https://github.com/terraform-google-modules/terraform-google-gcloud/pull/155

### Expected behavior

This param should've been added to the back so as not to break downstream packages.

### Observed behavior

It breaks and blocks a bunch of terraform pipelines

### Terraform Configuration

```hcl
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/3f27bc20bf5ee81ccda862307fa8c9624357ec93/modules/asm/main.tf#L58-L70
```

### Terraform Version

```sh
0.14 
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/3f27bc20bf5ee81ccda862307fa8c9624357ec93/modules/asm/versions.tf#L19
```

**Comment by apeabody** [comment]
Thanks for the report @joostschriek - I've opened https://github.com/terraform-google-modules/terraform-google-gcloud/pull/164

**Comment by apeabody** [comment]
Apologies @joostschriek - https://github.com/terraform-google-modules/terraform-google-gcloud/pull/155 has been reverted in https://github.com/terraform-google-modules/terraform-google-gcloud/releases/tag/v3.2.1.  If for some reason you are still seeing issues please reopen with the details.
